gateleen-security/src/test/resources/testresource_emptyproperties_mapper_resource

```diff
@@ -0,0 +1,8 @@
+{
+  "mappings": [{
+    "pattern":"",
+    "role":"",
+    "keepOriginal":false
+    }
+  ]
+}
```

Comment:
Github mentions a missing final "newline" - probably not an issue

Reply:
no idea what is missing here.

Reply:
See [why-should-text-files-end-with-a-newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).